README.md

```diff
@@ -20,78 +20,214 @@ from django_enum_choices.fields import EnumChoiceField
 
 from enum import Enum
 
-class FooBarEnumeration(Enum):
-    FOO = 'foo'
-    BAR = 'bar'
+class MyEnum(Enum):
+    A = 'a'
+    B = 'b'
 
 
 class MyModel(models.Model):
-    foo_bar_field = EnumChoiceField(FooBarEnumeration)
+    enumerated_field = EnumChoiceField(enum_class=MyEnum)
```

Comment:
I'd remove the `enum_class` everywhere from the examples, since it makes it more verbose.